Makefile

```
@@ -153,6 +156,13 @@ jupytext--to-ipynb: ## convert remote md files into ipynb
 	bin/jupytext --to ipynb work/src/notebook/**/*.md
 .PHONY: jupytext--to-ipynb
 
+seed-api: ## seed the API database (static data)
+seed-api: data/irve-statique.csv
+	cat data/irve-statique.csv | \
+		$(COMPOSE) exec -T api pipenv run csvjson -I --stream | \
+		bin/qcc static bulk
```

I've tried the following to activate the "I don't give a fuck" mode:

```suggestion
		bin/qcc static bulk --ignore-errors
```

and I was only able to import 10490 statiques with success.

---

To be perfectly clear, since we send static entries by batch of 10 records, it does not mean that only 10490 records are valid, but rather that 1049 batches contains fully valid records.

---

Anyway it means that we need to perform extra work to clean up our "valid" dataset. Ping @loco-philippe :pray:

---

The differences between "static schema" and data model are as follow: 

| attribute            | Qualicharge       | static schema |
| :------------------- | ----------------- | --------------- |
| contact_amenageur    | EmailStr          | format email    |
| contact_operateur  (M)  | EmailStr          | format email    |
| telephone_operateur  | FrenchPhoneNumber | string          |
| nbre_pdc   (M)          | PositiveInt       | int             |
| date_maj   (M)          | PastDate          | date            |
| num_pdl              | "^\d{14}$"        | string          |
| date_mise_en_service | PastDate          | date            |
| puissance_nominale (M)  | PositiveFloat     | float           |

For those differences, we have to decide how to transform data. For example : 

- optional data -> not loaded
- nbre_pdc ->  not loaded (calculated)

If it concerns an attribute outside of this list, we have to discuss it with `transport` team to have correct data.